### DISCUSSION TITLE
Azure Support

### DISCUSSION THREAD

**Discussion by Nebula-Ninjas** [OpenCSPM/opencspm#182] [Q&A] [answered]
@joshlarsen 

Is there support for Azure?

If so, is it only available within the Enterprise version?

Thanks in advance,

William

**Comment by joshlarsen** [comment, marked as answer]
No support for Azure currently.

**Reply by sanzio21** [reply]
Hi @joshlarsen,
sorry, could you tell me if it is included in the roadmap?
Thank you and best regards.
